generator/views/glean_ping_view.py

```diff
@@ -52,7 +52,7 @@
 
 # List of labeled counter names for which a suggest explore should be generated.
 # Generating suggest explores for all labeled counters slows down Looker.
-SUGGESTS_FOR_LABELED_COUNTERS = {"metrics__labeled_counter__glean_error_invalid_label"}
```

Comment:
These suggest explores have not been used in the last year. We don't need to generate them